lesson01/assignment/index2.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style2.css">
+    <title>Assignment Lesson 1 | Array Methods</title>
+</head>
+<body>
+    <h1>Array Methods in the Browser</h1>
+    <form id="first-form">
+        <label>Type the first thing that come accross to your mind:</label>
+            <p>"Remember your word must contain at least 5 characters"</p>
+        <input type="text" class="entry">
+        <input type="submit" value="Click here! And see what happens">
+            <p>This is what you wrote<span></span></p>
+        <button id="slc1">Click</button>
+            <p id="slc"></p>
+    </form>
+
+    <form id="second-form">
+        <label>Seasons</label>
+            <button id="show-season">Click</button>
+            <p id="print-season">Lorem ipsum dolor sit.</p>     
+    </form>
+
+    <form id="third-form">
+        <label>Months</label>
+            <p id="month-array">This is messy</p>
+            <p>February, May, August, October, January, June, September, December, March, July, November, April</p>
+        <input type="submit" value="Fix this!">
+            <p id="print-month"></p>
+    </form>
+
+    <form id="fourth-form">
+        <h2>Fruits</h2>
+        <ul>
+            <li>Apples</li>
+            <li>Oranges</li>
+            <li>Kale</li>
+            <li>Berries</li>
+        </ul>
+        <label>Which of these items is not a fruit?</label>
+            <button id="print-fruit">Show</button>
+                <p id="result"></p>
+    </form>
+
+    <div class="remove-color">
+        <button id="myFunction3">Try It!</button>
+            <div id="myDiv" class="bg-red">Change the background color</div>
+    </div>
+
+    <form id="fith-form">
+        <label>What's the average of 100?</label>
+        <input type="submit" value="GO" id="go">
+        <p id="solution"></p>
+    </form>
+    
+    <script src="./script-2.js"></script>

```

Comment:
If the javascript is wrapped in a DOM loaded event listener, like it is in the provided script.js, it can be loaded in the head of the HTML. That's a more professional solution.